docs/analyzer.md

```diff
@@ -16,10 +16,11 @@ PM> Install-Package FakeItEasy.Analyzer
 
 ## Diagnostics
 
-The analyzer currently provides two diagnostics:
+The analyzer currently provides the following diagnostics:
 
-| Id             | Summary                               | Description                                                                                                                                           |
-|----------------|---------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
-| FakeItEasy0001 | Unused call specification             | Triggered when you specify a call but don't configure or assert it, making it a no-op.                                                                |
-| FakeItEasy0002 | Non-virtual member configuration      | Triggered when you try to configure a non-virtual member, which cannot be faked.                                                                      |
-| FakeItEasy0003 | Argument constraint outside call spec | Triggered when you try to use an [argument constraint](argument-constraints.md) outside of a [call specification](specifying-a-call-to-configure.md). |
+| Id             | Summary                                  | Description                                                                                                                                           |
+|----------------|------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| FakeItEasy0001 | Unused call specification                | Triggered when you specify a call but don't configure or assert it, making it a no-op.                                                                |
+| FakeItEasy0002 | Non-virtual member configuration         | Triggered when you try to configure a non-virtual member, which cannot be faked.                                                                      |
+| FakeItEasy0003 | Argument constraint outside call spec    | Triggered when you try to use an [argument constraint](argument-constraints.md) outside of a [call specification](specifying-a-call-to-configure.md). |
+| FakeItEasy0004 | Argument constraint nullability mismatch | Triggered when you use a non-nullable [argument constraint](argument-constraints.md) for a nullable parameter. Calls where the argument is null won't be matched. If this is intentional, consider using `A<T?>.That.IsNotNull()` instead. If it's not, make the argument constraint nullable (`A<T?>`). |

```

Comment:
I assumed so, since it is in markdown. I just tried it using mkdocs and the readthedocs theme and see

![image](https://user-images.githubusercontent.com/3275797/27954051-3cb2208c-62dc-11e7-985a-7e4bdf03cb43.png)

generated from

```html
<table>
  <tr>
    <td>Id</td>
    <td>Summary</td>
    <td>Description</td>
  </tr>
  <tr>
    <td>FakeItEasy0001</td>
    <td>Unused call specification</td>
    <td>Triggered when you specify a call but don't configure or assert it, making it a no-op.</td>
  </tr>
  <tr>
    <td>FakeItEasy0002</td>
    <td>Non-virtual member configuration</td>
    <td>Triggered when you try to configure a non-virtual member, which cannot be faked.</td>
  </tr>
  <tr>
    <td>FakeItEasy0003</td>
    <td>Argument constraint outside call spec</td>
    <td>
        Triggered when you try to use an <a href="../argument-constraints">argument constraint</a> outside of a
        <a href="../specifying-a-call-to-configure">call specification</a>.
    </td>
  </tr>
  <tr>
    <td>FakeItEasy0004</td>
    <td>Argument constraint nullability mismatch</td>
    <td>
      Triggered when you use a non-nullable [argument constraint](argument-constraints.md) for a nullable parameter. 
      Calls where the argument is null won't be matched. If this is intentional, consider using <code>A&lt;T?&gt;.That.IsNotNull()</code> instead.
      If it's not, make the argument constraint nullable (<code>A&lt;T?&gt;</code>).
    </td>
  </tr>
</table>
```

Note that I did have to replace all the markdown _inside_ the table with HTML. We may be better off just sticking with the very long line.